pkg/builder/backup_builder.go

```diff
@@ -77,7 +80,23 @@ func (b *BackupBuilder) ObjectMeta(opts ...ObjectMetaOpt) *BackupBuilder {
 
 // FromSchedule sets the Backup's spec and labels from the Schedule template
 func (b *BackupBuilder) FromSchedule(schedule *velerov1api.Schedule) *BackupBuilder {
-	labels := schedule.Labels
+	var labels = make(map[string]string)
+
+	// Check if there's explicit Labels defined in the Schedule object template
+	// and if present then copy it to the backup object.
+	if schedule.Spec.Template.Metadata.Labels != nil {
+		labels = schedule.Spec.Template.Metadata.Labels
+		logrus.WithFields(logrus.Fields{
+			"backup": fmt.Sprintf("%s/%s", b.object.GetNamespace(), b.object.GetName()),
+			"labels": schedule.Spec.Template.Metadata.Labels,
+		}).Debug("Schedule.template.metadata.labels set - using those labels instead of schedule.labels for backup object")
```

Comment:
Let's make this Info level please, as normally Debug messages aren't output and this is a good heads up for when people start complaining about not getting the labels they expected on the backup

Reply:
Done. Nobody should be surprised - as they wouldn't have set template.metadata.labels normally (current schedule object does not allow it) :)

Reply:
They might have already had labels set on the schedule that were getting passed through and then they set the new field and then they start asking "Why didn't the labels on the schedule get passed through too".  Thanks!